lib/modules/manager/bazel/extract.spec.ts

```diff
@@ -1,7 +1,8 @@
 import { Fixtures } from '../../../../test/fixtures';
-import { extractPackageFile as extract } from '.';
+import { extractPackageFile as _extractPackageFile } from '.';
```

Comment:
why this rename?

Reply:
This is for less confusion, since we have another `extract` now